mne/io/pick.py

```diff
@@ -60,7 +60,11 @@ def channel_type(info, idx):
     elif kind == FIFF.FIFFV_SYST_CH:
         return 'syst'
     elif kind in [FIFF.FIFFV_SEEG_CH, 702]:  # 702 was used before v0.11
```

Comment:
Looks like it's in 0.11:

https://github.com/mne-tools/mne-python/blob/maint/0.11/mne/viz/raw.py#L249

?